decidim_app-design/app/views/admin/partials/_logs_list.html.erb

```diff
@@ -0,0 +1,90 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title">Admin logs</h2>
+  </div>
+  <div class="table-scroll">
+    <ul id="logs" class="table">
+      <% 5.times do |index| %>
+        <li class="logs-log">
```

Comment:
This should probably be `logs__log` as it's an element that needs to exist within a `logs` parent element.